litr/src/main/java/com/linkedin/android/litr/TransformationJob.java

```diff
@@ -245,10 +245,20 @@ void release(boolean success) {
             mediaTargets.add(trackTransform.getMediaTarget());
         }
         for (MediaSource mediaSource : mediaSources) {
-            mediaSource.release();
+            try {
+                // Sometimes this method will throw an exception, so we just need to catch it.
+                mediaSource.release();
```

Comment:
So I presume that if we catch this exception transformation completes and resulting transcoded video/audio is playable? 
Did you check what root cause of this might be?

Reply:
In my case, when the InputBuffer's size is larger than the OutputBuffer's size, and try to write this buffer to the OutputBuffer, then this will throw a `BufferOverflowException`, and this exception will be caught in TransformationJob in the `run()` method, but when trying to call the `error()` method then `release()` method, the `mediaTarget.release()` will throw an exception but not caught.

Reply:
Ok, great, currently I'll remove this change from the pull request, and let you make the proper fix for any exception that could be thrown from `mediaTarget.release()`